lib/Target/Jaxite/JaxiteEmitter.cpp

```diff
@@ -229,6 +246,35 @@ LogicalResult JaxiteEmitter::printOperation(tensor::FromElementsOp op) {
   return success();
 }
 
+// Allocating output ciphertexts.
+// Example: temp_nodes[idx] = [types.LweCiphertext]*size
+LogicalResult JaxiteEmitter::printOperation(memref::AllocOp op) {
+  emitAssignPrefix(op.getResult());
+  os << "[";
+  if (failed(emitType(op.getMemref().getType().getElementType()))) {
+    return op.emitOpError() << "Failed to get memref element type";
+  }
+  os << "]*" << op.getMemref().getType().getNumElements();
+  os << "\n";
+  return success();
+}
+
+// Assuming StoreOp is only used while storing results.
+// Example: temp_nodes[result_idx][idx] = temp_nodes[i]
+LogicalResult JaxiteEmitter::printOperation(memref::StoreOp op) {
+  os << "temp_nodes[" << variableNames->getIntForValue(op.getMemref()) << "]";
```

Comment:
TBH, kept the first thing that got the end-to-end working for add_one. Let me think about this a bit more. (It could be either 2D array as part of emitAssignPrefix() or a separate output variable.

I will resolve this comment after adding the changes.

Reply:
Added bracketEnclosedValues. Leaving the variable name as temp_nodes for now. 

Note to future self: Renaming it as a separate output variable doesn't work in the yosys modules case.